### ISSUE TITLE
Android app crashes my phone

### ISSUE THREAD

**Issue by lguerard** [rivafarabi/deckboard#320]
Hi,

I really like Deckboard but for a couple of weeks, every time I use Deckboard and shut the screen of my phone, I can't get the screen back after and I need to reboot the phone.

I tried without Deckboard and everything's fine, but it's really when Deckboard is open/is the latest app I used.

I'm on a OnePlus 6T and it happens both on A9 and A10 that I got lately.

Am I the only one ? Is there anything I can do to avoid that ?

Thanks !

**Comment by rivafarabi** [comment]
Sorry for the inconvenience. This is the first time this kind of issue has been reported. 

Is the crash happen when the app first opened or when first connected to your PC?

**Comment by lguerard** [comment]
The PC is the first connected since it starts with the computer. I then start the android app and link to the computer whenever I need to do anything.

If there are any logs on the phone that could help, let me know where and I can copy/paste them here.

**Comment by rivafarabi** [comment]
So the crash happens after it connected to PC? Noted. 

Btw which version are you using? Free or Pro version?

**Comment by lguerard** [comment]
Yeah so PC starts, I set up everything there, then I connect to the PC on the phone, use Deckboard, everything's fine but when I leave my phone and the screen goes black, no way on waking it up again :/.

As long as I know, I'm using the Free version.
